includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -352,12 +352,14 @@ protected function get_filtered_term_product_counts( $term_ids, $taxonomy, $quer
 		$tax_query      = new WP_Tax_Query( $tax_query );
 		$meta_query_sql = $meta_query->get_sql( 'post', $wpdb->posts, 'ID' );
 		$tax_query_sql  = $tax_query->get_sql( $wpdb->posts, 'ID' );
+		$post_type      = !is_array( $wp_the_query->query_vars['post_type'] ) ? array( $wp_the_query->query_vars['post_type'] ) : $wp_the_query->query_vars['post_type'];
+		$where          = apply_filters('woocommerce_filtered_term_product_counts_where_clause', "WHERE {$wpdb->posts}.post_type IN ('" . implode("','", array_map( 'esc_sql', $post_type )) . "') AND {$wpdb->posts}.post_status = 'publish'");
```

Comment:
Use woocommerce_get_filtered_term_product_counts_query_where, and pass $term_ids taxonomy and query type to it as arguments.

Brackets should also have a single space around them.